crates/pop-contracts/Cargo.toml

```diff
@@ -36,7 +36,6 @@ contract-extrinsics.workspace = true
 
 #  pop
 pop-common = { path = "../pop-common", version = "0.2.0" }
-pop-parachains = { path = "../pop-parachains", version = "0.2.0" }
```

Comment:
wtf

Reply:
This was an intermediate implementation that I used in PR 254.